tox.ini

```diff
@@ -4,40 +4,29 @@
 # and then run "tox" from this directory.
```

Comment:
not too familiar with tox files but i had to update to remove nose references and suppress unwanted warnings (W503 and W504 are conflicting warnings so you have to disable one or the other)

Reply:
also removed references to versions of python we don't care about.